README.md

```diff
@@ -291,7 +291,10 @@ These variables can be set in the `datadog_agent` class to control settings in t
 | `local_tags`                | An array of <KEY:VALUE> strings that are set as tags for the node.                                                                                                                               |
 | `non_local_traffic`         | Set this to allow other nodes to relay their traffic through this one.                                                                                                                           |
 | `agent5_enable`             | A boolean to install Agent v5 and override the Agent v6 default.                                                                                                                                 |
-| `apm_enabled`               | A boolean to enable the APM Agent (defaults to false).                                                                                                                                           |
+| `apm_enabled`               | A boolean to enable the APM Agent (defaults to false).                                                                                                         
+
+| `apm_trace_search_analytics_events`    | An array to add APM events for the Trace Search & Analytics tool. (defaults to []).
```

Comment:
Two suggestions to stay close the the underlying data structure used in the Agent6:

- Use a map structure. Also provide an example in the documentation:
```
{
  'my-rails-app|rails.request': 1,
  'service-name|operation-name': 1
}
```
- Simply call the variable `apm_analyzed_spans`?